tests/account_mt5_test.py

```diff
@@ -0,0 +1,47 @@
+from collections import namedtuple
+from mock import patch
+from src.account_mt5 import AccountMT5
+
+import pytest
```

Comment:
Why are we using two different types of testing frameworks? Wouldn't it be preferred to stick to one - either pytest or unittest?